app/assets/stylesheets/_screen.scss

```diff
@@ -641,12 +641,54 @@ div#keycut-help {
   }
 }
 
-.teams {
-  .card-link {
+.teams, .projects {
+  .card-link, .col-md-6 {
```

Comment:
could you add another class instead of modifying bootstrap's classes/behaviours here?